src/main/webapp/settings.js

```diff
@@ -1,3 +1,3 @@
-var settings = { // eslint-disable-line no-unused-vars
-    showHeader: false
+var defaultSettings = { // eslint-disable-line no-unused-vars
+    topBar: 'default' //possible values ['default', 'off', 'my custom headline']
```

Comment:
Will look into this, but i guess it should be 'off' by default (matches showHeader=false then).